src/services/analytics/gtm.ts

```diff
@@ -141,13 +141,15 @@ export const gtmTrackPageview = (pagePath: string): void => {
   gtmSend(gtmEvent)
 }
 
+const stripQueryStringAndHashFromPath = (url: string) => url.split('?')[0].split('#')[0]
```

Comment:
```suggestion
const normalizeAppName = (appName?: string): string => {
  // App name is a URL
  if (appName?.startsWith('http')) {
    // Strip search query and hash
    return appName.split('?')[0].split('#')[0]
  }
  return appName || ''
}
```

Worth adding some tests?

Reply:
done